src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+@use "../utils/_variables" as *;
+
+.main {
+  display: flex;
+  max-width: 944px;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  padding-top: 50px;
+  margin: 0 auto;
+}
+
+.card {
+  background: $clear_white-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &--position {
+    display: flex;
+    flex-direction: column;
+    padding: 32px 16px 16px;
+    max-width: 200px;
+    margin-bottom: 46px;
+  }
+
+  &__logo {
+    align-self: center;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__description {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: $dark_blue-for-text;
+  }
+
+  &__code {
+    color: $gray-for-text;
+    font-size: 10px;
+    margin-bottom: 16px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__raiting {
+    display: flex;
+    justify-content: space-between;
```

Comment:
Maybe it's better to make a mixin for those attributes, You use them a few times

Reply:
good idea, you can try it out